process.sh

```diff
@@ -15,7 +15,7 @@ main() {
     INPUT="$FILES/input"
     OUTPUT="$FILES/output"
 
-    for DATASET in linnarsson dries wang cao spraggins; do
+    for DATASET in linnarsson dries wang cao spraggins vanderbilt_MxIF; do
```

Comment:
Slight preference to just call it `vanderbilt`... unless you think we'll have `vanderbilt_Something-else` in the near future?

Reply:
Sure, I can change this.